media/css/firefox/new.less

```diff
@@ -341,7 +341,8 @@ html[lang|="en"] #main-headline span:first-child {
     width: 100%;
 }
 
-.download-button-wrapper.android {
+.download-button-wrapper.android,
+.download-button-wrapper.ios {
```

Comment:
Doh, I knew I was forgetting something. With all the branch-juggling I lost track of what was left to do in which. I'll add the CSS to display the button (remembering that the button won't actually display unless you have the updated button from the other branch, but you can also just switch it to android for local testing).

As for versions, I'm not sure we can actually determine latest version of FxiOS yet. It's on a different version number than the other Firefoxes and I don't know what's involved in getting "1.1" to be treated as latest, but only for FxiOS... that may take some extra work on the product details backend and I'm not sure what's in there already at this stage. I'll ask, but I think for this release we may not be able to do any special "you're up to date" or "you need a refresh" logic, that may have to come later.